cdk/domino_cdk/domino_stack.py

```diff
@@ -147,24 +146,4 @@ def generate_outputs(self):
                 value=r53_owner_id,
             )
 
-        if self.cfg.install is not None:
-            agent_cfg = generate_install_config(
```

Comment:
If we outright remove the template, we should also rewrite this:

https://docs.dominodatalab.com/en/latest/admin_guide/c1eec3/deploy-domino/#_configure_the_cluster